index.bs

```diff
@@ -670,6 +686,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
     </div>
     </li>
 
+    <li><dfn>Focus distance</dfn> is a numeric camera setting that controls the focus distance of the lens.  The setting usually represents distance in millimeters to the optimal focus distance.</li>
```

Comment:
According to the mediacapture spec, we [should](https://w3c.github.io/mediacapture-main/getusermedia.html#dom-constrainablepattern-getcapabilities())  normalize hardware settings.